lib/3llo/command/board.rb

```diff
@@ -21,9 +21,7 @@ def execute(subcommand, args)
         else
           handle_invalid_subcommand(subcommand, args)
         end
-      rescue InvalidCommandError => exception
-        Command::Board::Invalid.execute(exception.message)
-      rescue InvalidArgumentError => exception
+      rescue InvalidCommandError, InvalidArgumentError, URI::InvalidURIError => exception
```

Comment:
I think I would handle the invalid ID here instead.

https://github.com/qcam/3llo/blob/master/lib/3llo/command/board/select.rb#L24

Reply:
It would be more effective when we handle [here](https://github.com/qcam/3llo/blob/master/lib/3llo/command.rb#L68) because I realized some commands that take the ID argument will cause crashes.

Reply:
In that case, I think we should handle the exception at where it was raised: https://github.com/qcam/3llo/blob/master/lib/3llo/remote_server.rb#L80.

Reply:
Nice, I've fixed using Regexp, please check it out.